cpp/data/email_watcher.conf

```diff
@@ -0,0 +1,26 @@
+# Each section must provide the following entries:
+# from_host                    -- where the emails are originating
+# subject_pattern              -- a PCRE regex that must match the emails' subjects
+# email_body_positive_pattern  -- a PCRE that must match the emails' bodies
```

Comment:
Same as variables:
One pattern is called "subject_pattern" so the others should be called e.g "body_positive_pattern" instead of "email_body_positive pattern"